queries/dart/highlights.scm

```diff
@@ -90,7 +87,6 @@
   name: (identifier) @type)
 (enum_constant
   name: (identifier) @type)
-(type_identifier) @type
```

Comment:
Why is `type_identifier` not needed anymore?

Reply:
It was a duplicate, the exact same thing is present at line 97. There was a bit of messiness generally so some of the other changes are tidying up.